README.md

```diff
@@ -46,50 +43,62 @@ The output should contain something like this `/home/user/lava`
 
 #### [Linux/MacOS]
 ```bash
-$ git clone git@github.com:lava-dl/lava.git
-$ cd lava-dl
-$ pip install -r build-requirements.txt
-$ pip install -r requirements.txt
-$ export PYTHONPATH=$PYTHONPATH:$(pwd)/src
-$ pyb -E unit
+cd $HOME
+git clone git@github.com:lava-dl/lava.git
```

Comment:
This should be `lava-dl.git` not `lava.git` but it looks like this has been present for some time.

Reply:
Fixed.